bind_exporter.go

```diff
@@ -205,6 +205,11 @@ var (
 		"Total number of available worker threads.",
 		nil, nil,
 	)
+	zoneSerial = prometheus.NewDesc(
+		prometheus.BuildFQName(namespace, "", "zone_serial"),
+		"Zone serial number.",
+		[]string{"view", "name"}, nil,
```

Comment:
Minor nit, `name` is a bit generic, maybe `zone` or `zone_name` to make it more specific and less likely to conflict with other (discovery) labels?

Reply:
Fixed. 👍